gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/QueryServiceRequestAndResponseUtils.java

```diff
@@ -162,29 +175,31 @@ public static Filter createQsDefaultRequestFilter(String timestampColumnName,
                                                     long startTime,
                                                     long endTime) {
     return Filter.newBuilder()
-        .addChildFilter(
-            createQsFilter(
-                createQsColumnExpression(timestampColumnName),
-                Operator.GE,
-                createQsStringLiteralExpression(Long.toString(startTime))
-            )
-        )
-        .addChildFilter(
-            createQsFilter(
-                createQsColumnExpression(timestampColumnName),
-                Operator.LT,
-                createQsStringLiteralExpression(Long.toString(endTime))
-            )
-        )
+        .setOperator(Operator.AND)
         .addChildFilter(
             Filter.newBuilder()
+                .setOperator(Operator.AND)
                 .addChildFilter(
                     createQsFilter(
-                        createQsColumnExpression(entityIdColumnName),
-                        Operator.NEQ,
-                        createQsStringLiteralExpression("null")
+                        createQsColumnExpression(timestampColumnName),
+                        Operator.GE,
+                        createQsLongLiteralExpression(startTime)
```

Comment:
question: QS now works nicely with types?

Reply:
Oh gosh.. I didn't know that but would verify that it does otherwise i'll have to revert this change :)

Reply:
I can't remember either, but looking at the old filter building code, it was working in strings so I'd suspect we'd need to do the same.

Reply:
I've tested this and it works fine.